satpy/readers/native_msg.py

```diff
@@ -59,7 +58,7 @@ class NativeMSGFileHandler(BaseFileHandler, SEVIRICalibrationHandler):
     """SEVIRI native format reader.
     """
 
-    def __init__(self, filename, filename_info, filetype_info):
+    def __init__(self, filename, filename_info, filetype_info, **reader_kwargs):
```

Comment:
Please specify `calibration_mode='Nominal'` as an explicit keyword argument.

Reply:
If the 'Nominal' doesn't mean something specifically in the file format then it may be more pythonic to be all lowercase too.

Reply:
hi. i dont quite undertsand your first point. We dont call(initialise)  the Filehandlers with CalibrationMode explicitly so dont know how this would work.

Reply:
No, but you pass it anyway, so having it here should work

Reply:
@ColinDuff If you do:

```python
def my_func(pos1, pos2, kw1=1, kw2=2):
    pass

kwargs = {'kw1': 5}
my_func(1, 2, **kwargs)
```

It will expand the kwargs and pass them each individually. The benefit of defining the function/method this way is that the code shows exactly what the keyword argument is in the definition and what its default is.

Reply:
ok , understood